utils/validators.ts

```diff
@@ -8,18 +8,44 @@ const { isAddress } = web3utils // need to import like this due to CommonJS modu
  * @param value - the value to check
  * @returns - throws an error if the value is not an address
  */
-export function assertAddress(
-  value?: string | string[]
-): asserts value is Address {
+export function assertAddress(value?: string): asserts value is Address {
   if (!value) {
     throw Error(`missing address`)
```

Comment:
Maybe we can be more specific for this error and specify which address, in what context so it s easier to debug afterwards :)